client/src/components/AlumniDirectory.js

```diff
@@ -267,6 +310,14 @@ export default class AlumniDirectory extends Component {
                 
                 {searchRow}
                 {resultsRow}
+                {this.state.requestModalOpen && 
```

Comment:
Do we need the `this.state.requestModalOpen && `?

Reply:
Yes - otherwise it will try to render without an alumni prop